test/method/testddec.py

```diff
@@ -83,15 +83,20 @@ def test_proatom_read(self):
         assert_allclose(self.analysis.proatom_density[1][0:5], refH_den, rtol=1e-3)
         assert_allclose(self.analysis.proatom_density[2][0:5], refH_den, rtol=1e-3)
 
-    def test_step1_charges(self):
-        """Are step 1 charges calculated correctly?"""
+    def test_step1_and_2_charges(self):
+        """Are step 1 and 2 charges calculated correctly?"""
 
         self.parse()
         # use precalculated fine cube file
-        imported_vol = volume.read_from_cube(os.path.join(os.path.dirname(os.path.realpath(__file__)), "water_fine.cube"))
-        
+        imported_vol = volume.read_from_cube(
+            os.path.join(os.path.dirname(os.path.realpath(__file__)), "water_fine.cube")
+        )
+
         analysis = DDEC6(self.data, imported_vol, os.path.dirname(os.path.realpath(__file__)))
         analysis.calculate()
-        
+
         # values from `chargemol` calculation
-        assert_allclose(analysis.refcharges, [-0.513006, 0.256231, 0.256775], rtol=.1)
+        # which is based on proatomic densities calculated with different basis set.
+        # (rtol is adjusted to account for this inevitable discrepancy)
+        assert_allclose(analysis.refcharges[0], [-0.513006, 0.256231, 0.256775], rtol=0.10)
```

Comment:
Were you going to add Horton values as well? And comment on where you think the chargemol discrepancy comes from?

Reply:
I was thinking of adding `horton` values after full implementation when I add other simple molecules into the test suite. It looks like I've added the comments about the discrepancy only on the Step 3 PR. I've included them here as well. Thanks!